tests/containers/suma_containers.pm

```diff
@@ -0,0 +1,49 @@
+
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Test SUMA container images:
+#   The scope of the testing is minimal and covers pulling and running 
+#   container images from the internal registry.suse.de
+#   
+# The container images tested are:
+#   - proxy-httpd
+#   - proxy-salt-broker
+#   - proxy-squid
+#   - proxy-ssh
+#   - proxy-tftpd
+#
+# The testing of proxy-charts is limited to pulling the tar from the registry to
+# and inspect the content to ensure Charts is available. 
+# Install or deployment is not covered in this scenario.
+#   
+# In depth testing of all the images  is already done by the SUMA QE team
+#
+# Maintainer: Maurizio Galli <maurizio.galli@suse.com>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use serial_terminal 'select_serial_terminal';
+use utils;
+
+sub run {
+    select_serial_terminal;
+
+    my $image = get_required_var('CONTAINER_IMAGE_TO_TEST');
+
+    if (check_var('CONTAINER_SUMA', 'image')) {
+        my $runtime = get_required_var('CONTAINER_RUNTIME');
+        script_retry("for i in $(echo $runtime | tr ',' '\n'); do $i pull $image; done", timeout => 300, delay => 60, retry => 3);
```

Comment:
```suggestion
        my @runtime = split /,/, $runtime;
        for $runtime (@runtime) {
            script_retry("$runtime pull $image", timeout => 300, delay => 60, retry => 3);
            assert_script_run("$runtime run --rm $image true");
        }
```

What do you think of this approach instead of using bash `for` loop?

Reply:
With this and a rebase I think we're done. If you agree, please update the PR and ping me for verification runs, then we can proceed and merge it.

Reply:
well I did it in bash because that's what I know, if you prefer to have it in Perl I have no objections, I just don't really know how to write Perl code ;)

Reply:
Understandable :slightly_smiling_face: